Neos.Neos/Classes/Fusion/Cache/CacheTag.php

```diff
@@ -32,7 +32,7 @@ private function __construct(
 
     final public static function forNodeAggregate(
         ContentRepositoryId $contentRepositoryId,
-        WorkspaceName $workspaceName,
+        ?WorkspaceName $workspaceName,
```

Comment:
I'm always a bit suspicious when it comes to nullable method arguments because that is a bit error prone (e.g. `CacheTag::forNodeAggregate($crId, $workspaceName, $nodeId)` will not fail if `$workspaceName` is null for some reason – and that can happen quickly, especially in the context of Fusion)
So I would suggest to...